conanfile.py

```diff
@@ -48,7 +49,8 @@ class ImguiSFML(ConanFile):
         'imconfig': None,
         'imconfig_install_folder': None
     }
-    exports_sources = ['*.cpp', '*.h', 'CMakeLists.txt', 'cmake*']
+    exports_sources = ['imgui-SFML.cpp', 'imgui-SFML.h', 'imconfig-SFML.h',
```

Comment:
Minor, but let's not do it in one line.
One line per file, please. Will be easier for git later.